mdaviz/mainwindow.py

```diff
@@ -19,36 +21,41 @@ def __init__(self):
         self.setup()
 
     def setup(self):
-        self._folderPath = None
-        self._folderName = None
-        ###
-        self._folderList = None
-        self._mdaFileName = None
-        ###
-        self._folderSize = None
-        self._mdaFileList = None
+        self._dataPath = None       # the combined data path obj (folder.parent + subfolder)
+        self._folderPath = None     # the path obj from pull down 1
+        self._folderList = None     # the list of folder in pull down 1
+        self._subFolderPath = None  # the subfolder path obj selected in pull down 2
+        self._subFolderList = None  # the list of subfolder in pull down 2
+        self._mdaFileList = None    # the list of mda file NAME str (name only)
+        self._mdaFileLen = None     # the number of mda files in the list
```

Comment:
Why not define to be 0 instead?  Also, consider renaming `mdaFileLen` to `mdaFileCount`.  Seems more precise as to what this number describes.